### DISCUSSION TITLE
sqlpackage.exe ignore error parameter

### DISCUSSION THREAD

**Discussion by semics-tech** [microsoft/DacFx#371] [Q&A] [answered]
I am using the sqlpackage.exe cli tool to deploy databases from dacpac's but I can't find anywhere how to ignore errors on publish. I wouldn't usually do this but I have an edge case where I just need the database to publish for testing and the ignored objects are not needed.

I can see that if I generate the script, there is a line which I can manually change which does this but is there a paramter I can give to sqlpackage to change this?

If I change this line

```sql
GO
:on error exit
GO
```

to this 

```sql
GO
:on error ignore
GO
```

I get the desired output.

**Comment by asrichesson** [comment, marked as answer]
Kinda related to #306

**Reply by semics-tech** [reply]
I had seen this before and tried but thought it didn't work as it adds `:on error ignore` to the script further down rather than replacing the original `:on error exit`. I can see now that this works, thanks!
